kibana/wedeploy.json

```diff
@@ -3,12 +3,15 @@
     "image": "docker.elastic.co/kibana/kibana:6.2.4",
     "port": 5601,
     "cpu": 2,
-    "memory": 1024,
+    "memory": 2048,
```

Comment:
Not sure it's necessary.

Reply:
it's working fine with 1gb, look: https://console.wedeploy.com/projects/kibanawehealth/services

Reply:
So you will revert it back to 1GB?